src/molecules/form-fields/text-area-form-field.test.tsx

```diff
@@ -16,4 +19,95 @@ describe("TextAreaFormField", () => {
         // Assert
         expect(getByLabelText(expected)).not.toBeNull();
     });
+
+    test("when has errorMessages prop, renders text area form field with error message", () => {
```

Comment:
suggestion: being this test is around passing in an array of messages, expand your arrangement to have at least two values passed into the `errorMessages` prop and pluralize the test name above

Reply:
Thanks for catching those. I have updated the PR to reflect those changes.